tests/integration_tests/functional/test_balloon.py

```diff
@@ -196,58 +199,18 @@ def test_deflate_on_oom_true(test_microvm_with_api, network_config):
     # Note that due to `test_deflate_on_oom_false`, we know that
     # if `deflate_on_oom` were set to False, then such an error
     # would have happened.
-    make_guest_dirty_memory(ssh_connection)
-
-
-# pylint: disable=C0103
-def test_deflate_on_oom_false(test_microvm_with_api, network_config):
-    """
-    Verify that setting the `deflate_on_oom` to False works correctly.
-    """
-    test_microvm = test_microvm_with_api
-    test_microvm.spawn()
-    test_microvm.basic_config()
-    _tap, _, _ = test_microvm.ssh_network_config(network_config, "1")
-
-    # Add a memory balloon.
-    response = test_microvm.balloon.put(
-        amount_mib=0, deflate_on_oom=False, stats_polling_interval_s=0
-    )
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
-
-    # Start the microvm.
-    test_microvm.start()
-
-    # Get an ssh connection to the microvm.
-    firecracker_pid = test_microvm.jailer_clone_pid
-    ssh_connection = test_microvm.ssh
-
-    # We get an initial reading of the RSS, then calculate the amount
-    # we need to inflate the balloon with by subtracting it from the
-    # VM size and adding an offset of 10 MiB in order to make sure we
-    # get a lower reading than the initial one.
-    initial_rss = get_stable_rss_mem_by_pid(firecracker_pid)
-    inflate_size = 256 - int(initial_rss / 1024) + 10
-
-    # Inflate the balloon.
-    response = test_microvm.balloon.patch(amount_mib=inflate_size)
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
-    # This call will internally wait for rss to become stable.
-    _ = get_stable_rss_mem_by_pid(firecracker_pid)
-
-    # Check that using memory does lead to an out of memory error.
-    make_guest_dirty_memory(ssh_connection, should_oom=True)
```

Comment:
Changes unrelated to SSH stuff are included.

Reply:
fixed it on my forked repo by creating a separate commit: https://github.com/zulinx86/firecracker/commits/new-ci-artifacts
- https://github.com/firecracker-microvm/firecracker/commit/9bd10e662a7764b51353673b138c4ddbb75f589d

does this legit look to you?